### PR TITLE
VIH-9306 add justice user

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-9306

### Change description ###

* Add justice user form

https://user-images.githubusercontent.com/41630528/221170791-263f65a0-1efd-4430-9fc3-1acfa29e09c6.mov

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/common/custom-validations/email-validator.ts [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+    const valid = isAValidEmail(control.value);
     if (!valid) {
         return { validEmail: true };
     }
     return null;
 }
+
+export function isAValidEmail(input: string) {
+    /* tslint:disable: max-line-length */
+    const pattern = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (phillipjchambers) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/common/custom-validations/email-validator.ts:13]: might want to chat with Rob about this one

**Review by oliver-scott — APPROVED** [review]

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/justice-user-form/justice-user-form.component.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      <p *ngIf="form.controls.contactTelephone.invalid" id="contactTelephone-error" class="govuk-error-message">
+        <span class="govuk-visually-hidden">Error:</span> Enter the contact telephone
+      </p>
+      <input
+        class="govuk-input"
+        formControlName="contactTelephone"
+        id="contactTelephone"
+        name="contactTelephone"
+        type="text"
+        autocomplete="email"
```
Comment (oliver-scott) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/justice-user-form/justice-user-form.component.html:62]: maybe this should be `autocomplete="tel"`?

**Review by oliver-scott — COMMENTED** [review]
looks good. Just one minor observation

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/justice-user-form/justice-user-form.component.html (hunk 2 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/justice-user-form/justice-user-form.component.html:62]: ah you're right, I forgot to swap it when I removed contact email from the form

**Review by phillipjchambers — APPROVED** [review]

**Review by oliver-scott — APPROVED** [review]

**Pull request merged**